test/Visualization_Tests/src/Table_Spec.enso

```diff
@@ -10,22 +10,43 @@ import Standard.Visualization.Table.Visualization
 import Standard.Visualization.Helpers
 import Standard.Visualization.Id.Id
 
-from Standard.Test import Test, Test_Suite
-import Standard.Test.Extensions
+from Standard.Test_New import all
+
 import Standard.Base.Errors.Common.Type_Error
 
 polyglot java import java.util.UUID
 
+type Data
+    Value ~data
+
+    connection self = self.data.at 0
+    t self = self.data.at 1
+    t2 self = self.data.at 2
+
+    setup = Data.Value <|
+        enso_project.data.create_directory
+        file = enso_project.data / "sqlite_test.db"
+        file.delete_if_exists
+        connection = Database.connect (SQLite file)
+        in_mem = Table.new [["A", ['a', 'a', 'a']], ["B", [2, 2, 3]], ["C", [3, 5, 6]]]
+        t = in_mem.select_into_database_table connection "T" primary_key=Nothing temporary=True
+        t2 = Table.new [["A", [1, 2, 3]], ["B", [4, 5, 6]], ["C", [7, 8, 9]]]
+        [connection, t, t2]
+
+    teardown self =
+        self.connection.close
+        file = enso_project.data / "sqlite_test.db"
+        file.delete_if_exists
```

Comment:
I know you ported existing code and this is a good 1-1 port of it.

But looking at it - I highly doubt we actually need a file connection here at all.

I think we could be able to completely remove the setup and teardown and just rely on good old `Database.connect (SQLite In_Memory)`.

Would you be willing to try it?